dapps/react-dapp-v2/src/pages/index.tsx

```diff
@@ -108,6 +108,18 @@ const Home: NextPage = () => {
     await ping();
   };
 
+  async function emit() {
+    if (typeof client === "undefined") {
+      throw new Error("WalletConnect is not initialized");
+    }
+    
+    await client.emit({
+      topic: session?.topic || '',
+      event: { name: 'chainChanged', data: 'Hello World' },
```

Comment:
Think it's better to not provide `data` than using filler values for it

```suggestion
      event: { name: 'chainChanged' },
```